Shared/mods/deathmatch/logic/luadefs/CLuaUtilDefs.cpp

```diff
@@ -46,6 +87,10 @@ void CLuaUtilDefs::LoadFunctions()
         // Utility functions
         {"gettok", GetTok},
         {"tocolor", tocolor},
+
+#ifdef MTA_CLIENT
+        {"getSystemInfo", ArgumentParser<mta::GetSystemInfo>}
```

Comment:
Missing trailing comma.
```suggestion
        {"getSystemInfo", ArgumentParser<mta::GetSystemInfo>},
```